src/CTA.WebForms/FileConverters/CodeFileConverter.cs

```diff
@@ -42,11 +42,11 @@ public class CodeFileConverter : FileConverter
             try
             {
                 _fileModel = _webFormsProjectAnaylzer.AnalyzerResult.ProjectBuildResult?.SourceFileBuildResults?
-                    .Single(r => r.SourceFileFullPath.EndsWith(RelativePath))?.SemanticModel;
+                    .Single(r => r.SourceFileFullPath.Equals(fullPath, StringComparison.OrdinalIgnoreCase))?.SemanticModel;
```

Comment:
Just a thought: I like specifying the StringComparer, but do we actually want to ignore case here? Linux file systems are generally case sensitive. That said, I don't know if we've ever run any of this stuff on Linux anyway and if a ton of other stuff would break anyway.

Reply:
For the most part we can assume that that this will be run in a Windows environment, but CTA can run on a Linux machine so I think it's an edge case we should be aware of. 

@saikiranakula-amzn What are your thoughts here?

Reply:
The paths we are looking here are location of source files in a project, which will follow a specific hierarchal structure, It doesn't matter whether the platform is either windows/linux